index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="normalize.css">
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js" defer></script>
+    <title>Jarver's Crypt Maze</title>
+</head>
+<body>
+  
+  <!-- splash screen -->
+          
+    <div class="splash-main">
+        <div class="splash-heading"><h1>Jarver's Crypt</h1></div>
+        <div class="splash-outerbox">
+            <div class="splash-innerbox">
+                <img src="download.png" class="game-logo">
+                <button class="start-button">Start</button>
+            </div>
+            <div class="splash-innerbox">
+                <h2 class="instructionsTitle">Instructions</h2>
+                <div class="instructionDetails">
+                    <p>Oh no! You've been buried alive inside Jarver's Crypt!</p>
+                    <p>Place your cursor on the coffin.</p>
+                    <p>The game will begin once you leave the coffin square and enter the maze.</p>
+                    <p>Navigate your escape along the path using your mouse. </p>
+                    <p>Don't let the wandering spirits grab you or you'll be trapped..... forever!</p>
+                </div>
+            </div>
+        </div>
+    </div>
+  
+  <!-- Game screen -->
+  <div id="gameScreen">
+    <h1>Jarver's Crypt</h1>
+    <div class="maze collision">
+        <div class="start_area">
+            <h2>Start</h2>
+            <img src="images/coffin.png" alt="a coffin">
+        </div>
+        <div class="line">
+            <div class="vertical line_one thick"></div>
+            <div class="horizontal line_two thick"></div>
+            <div class="vertical line_three thick"></div>
+            <div class="horizontal line_four thick"></div>
+            <div class="vertical line_five thick"></div>
+            <div class="horizontal line_six thin"></div>
+            <div class="vertical line_seven thin"></div>
+            <div class="horizontal line_eight thick"></div>
+            <div class="horizontal line_nine thick"></div>
+            <div class="vertical line_ten thick"></div>
+            <div class="horizontal line_eleven"></div>
+        </div>
+        <div class="finish_area">
```

Comment:
there is a problem with the door, when you get close to it, you stop being a ghost